src/client/components/Story/Body.js

```diff
@@ -122,9 +122,8 @@ Body.defaultProps = {
   exitPageSetting: true,
 };
 
-export default connect(
-  state => ({
-    exitPageSetting: getExitPageSetting(state),
-  }),
-  null,
-)(Body);
+const BodyContainer = props => <Body {...props} />;
+
+export default connect(state => ({
```

Comment:
I think we should separate dumb and container components. Right now it works the same way as when we connected to Body directly. Maybe we should finally create `containers` directory.

Reply:
Honestly I wouldn't call the Body component a dumb component. There are a lot of logic code for displaying data.

Reply:
I agree that there should be containers that handle state and other app logic and presentational components. But that would require a lot of work and it's not the point of the PR. 
What should be done precisely to finalize this PR ?

Reply:
I'd say presentational components encapsulate UI logic, which I think current `Body` component does. Containers would just provide data to presentational components.

`components` folder should not contain containers, so I would extract `BodyContainer` to `containers` folder and use it in the app.